apps/web/pages/auth/logout.tsx

```diff
@@ -51,6 +52,11 @@ export default function Logout(props: Props) {
 
 export async function getServerSideProps(context: GetServerSidePropsContext) {
   const ssr = await ssrInit(context);
+  // Deleting old cookie manually, remove this code after all existing cookies have expired
```

Comment:
How can we know when all cookies are expired btw? What is max current expiryDate of cookies we can calculate it using that maybe

Reply:
I suppose newly created ones, have the same time span configuration. We can take a look there.